src/RA_Interface.cpp

```diff
@@ -68,59 +68,40 @@ void RA_GetEstimatedGameTitle(char* sNameOut)
+bool    (CCONV *_RA_ConfirmLoadNewRom)(bool bQuitting) = nullptr;
+int     (CCONV *_RA_OnLoadNewRom)(const BYTE* pROM, unsigned int nROMSize) = nullptr;
+void    (CCONV *_RA_InstallMemoryBank)(int nBankID, void* pReader, void* pWriter, int nBankSize) = nullptr;
+void    (CCONV *_RA_ClearMemoryBanks)() = nullptr;
+void    (CCONV *_RA_OnLoadState)(const char* sFilename) = nullptr;
+void    (CCONV *_RA_OnSaveState)(const char* sFilename) = nullptr;
+void    (CCONV *_RA_OnReset)() = nullptr;
 //	Achievements:
-void	(CCONV *_RA_DoAchievementsFrame)() = nullptr;
+void    (CCONV *_RA_DoAchievementsFrame)() = nullptr;
 //	User:
-bool	(CCONV *_RA_UserLoggedIn)() = nullptr;
-const char*	(CCONV *_RA_Username)() = nullptr;
-void	(CCONV *_RA_AttemptLogin)(bool bBlocking) = nullptr;
-//	Graphics:
-void	(CCONV *_RA_InitDirectX) (void) = nullptr;
-void	(CCONV *_RA_OnPaint)(HWND hWnd) = nullptr;
+void    (CCONV *_RA_AttemptLogin)(bool bBlocking) = nullptr;
 //	Tools:
-void	(CCONV *_RA_SetPaused)(bool bIsPaused) = nullptr;
-HMENU(CCONV *_RA_CreatePopupMenu)() = nullptr;
-void	(CCONV *_RA_UpdateAppTitle) (const char* pMessage) = nullptr;
-void	(CCONV *_RA_HandleHTTPResults) (void) = nullptr;
-void	(CCONV *_RA_InvokeDialog)(LPARAM nID) = nullptr;
-void	(CCONV *_RA_InstallSharedFunctions)(bool(*)(), void(*)(), void(*)(), void(*)(), void(*)(char*), void(*)(), void(*)(const char*)) = nullptr;
-int		(CCONV *_RA_SetConsoleID)(unsigned int nConsoleID) = nullptr;
-int		(CCONV *_RA_HardcoreModeIsActive)(void) = nullptr;
-int		(CCONV *_RA_HTTPGetRequestExists)(const char* sPageName) = nullptr;
-
-//	Don't expose to app
-HINSTANCE g_hRADLL = nullptr;
-
-
-int		(CCONV *_RA_UpdateOverlay) (ControllerInput* pInput, float fDeltaTime, bool Full_Screen, bool Paused) = nullptr;
-int		(CCONV *_RA_UpdatePopups) (ControllerInput* pInput, float fDeltaTime, bool Full_Screen, bool Paused) = nullptr;
-void	(CCONV *_RA_RenderOverlay) (HDC hDC, RECT* prcSize) = nullptr;
-void	(CCONV *_RA_RenderPopups) (HDC hDC, RECT* prcSize) = nullptr;
+void    (CCONV *_RA_SetPaused)(bool bIsPaused) = nullptr;
+HMENU   (CCONV *_RA_CreatePopupMenu)() = nullptr;
+void    (CCONV *_RA_UpdateAppTitle)(const char* pMessage) = nullptr;
+void    (CCONV *_RA_HandleHTTPResults)(void) = nullptr;
+void    (CCONV *_RA_InvokeDialog)(LPARAM nID) = nullptr;
+void    (CCONV *_RA_InstallSharedFunctions)(bool(*)(), void(*)(), void(*)(), void(*)(), void(*)(char*), void(*)(), void(*)(const char*)) = nullptr;
+int     (CCONV *_RA_SetConsoleID)(unsigned int nConsoleID) = nullptr;
+int     (CCONV *_RA_HardcoreModeIsActive)(void) = nullptr;
+//  Overlay:
+int     (CCONV *_RA_UpdateOverlay)(ControllerInput* pInput, float fDeltaTime, bool Full_Screen, bool Paused) = nullptr;
+int     (CCONV *_RA_UpdatePopups)(ControllerInput* pInput, float fDeltaTime, bool Full_Screen, bool Paused) = nullptr;
+void    (CCONV *_RA_RenderOverlay)(HDC hDC, RECT* prcSize) = nullptr;
+void    (CCONV *_RA_RenderPopups)(HDC hDC, RECT* prcSize) = nullptr;
```

Comment:
these are just whitespace changes (tabs still existing between the return type and the pointer declaration.